docs/_static/copy.js

```diff
@@ -0,0 +1,22 @@
+const COPY = "Copy";
+const COPIED = "Copied!";
+
+const copy = async (obj) => {
+  await navigator.clipboard.writeText(obj.children[1].innerText);
```

Comment:
This can fail, so you should probably catch it `alert` here so the user isn't confused when the button doesn't do anything.